source/command-monitoring/tests/unified/deleteMany.yml

```diff
@@ -0,0 +1,79 @@
+description: "deleteMany"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client client
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+        - commandFailedEvent
+  - database:
+      id: &database database
+      client: *client
+      databaseName: &databaseName command-monitoring-tests
+  - collection:
+      id: &collection collection
+      database: *database
+      collectionName: &collectionName test
+
+initialData:
+  - collectionName: *collectionName
+    databaseName: *databaseName
+    documents:
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+
+tests:
+  - description: "A successful deleteMany"
+    operations:
+      - name: deleteMany
+        object: *collection
+        arguments:
+          filter: { _id: { $gt: 1 } }
+    expectEvents:
+      - client: *client
+        events:
+          - commandStartedEvent:
+              command:
+                delete: *collectionName
+                deletes:
+                  - { q: { _id: { $gt: 1 }}, limit: 0 }
+                ordered: true
+              commandName: delete
+              databaseName: *databaseName
+          - commandSucceededEvent:
+              reply: { ok: 1, n: 2 }
+              commandName: delete
+
+  - description: "A successful deleteMany with write errors"
+    operations:
+      - name: deleteMany
+        object: *collection
+        arguments:
+          filter: { _id: { $nothing: 1 } }
```

Comment:
Nit: there is prior art for `$unsupportedQueryOperator` and `$unsupported` if you think that'd be more self-documenting.

Reply:
nice idea, updated all to `$unsupported`.